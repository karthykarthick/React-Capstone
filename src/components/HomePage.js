import React from 'react';
import ConnectedMealsListFilter from '../containers/MealsListFilter';
import ConnectedMealsList from '../containers/MealsList';

const HomePage = () => (
  <div className="page-container">
    <ConnectedMealsListFilter />
    <ConnectedMealsList />
  </div>
);

export default HomePage;
