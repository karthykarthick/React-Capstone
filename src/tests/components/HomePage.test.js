import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import HomePage from '../../components/HomePage';

const renderer = new ShallowRenderer();

it('should render HomePage correctly', () => {
  renderer.render(<HomePage />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
