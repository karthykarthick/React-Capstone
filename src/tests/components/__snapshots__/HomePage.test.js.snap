// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<div
  className="page-container"
>
  <Memo(Connect(MealsListFilter)) />
  <Memo(Connect(MealsList)) />
</div>
`;
